lib/lightstep/tracer.rb

```diff
@@ -169,15 +182,17 @@ def configure(component_name:, access_token: nil, transport: nil)
 
     CARRIER_TRACER_STATE_PREFIX = 'ot-tracer-'.freeze
     CARRIER_BAGGAGE_PREFIX = 'ot-baggage-'.freeze
+    CARRIER_RACK_HTTP_TRACER_STATE_PREFIX = "#{CARRIER_TRACER_STATE_PREFIX.gsub("-", "_").upcase}"
```

Comment:
In order to interoperate with tracers in other languages, the http headers used by lightstep libs need to match (at least in the HTTP headers sense of "match"... case doesn't matter, etc). The `_`s here are thus concerning to me...